eks-anywhere-common/Addons/Partner/Pulumi/pulumi-tester-serviceaaccount.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
If this configmap is for tester, this should be moved to `Testers/Pulumi` folder.